tests/testthat/_snaps/tidyeval-across.md

```diff
@@ -36,18 +36,18 @@
       capture_across(lf, across(a, 1))
     Condition
       Error in `across_funs()`:
-      ! `.fns` argument to dbplyr::across() must be a NULL, a function, formula, or list
```

Comment:
In a future PR, would be good to think about how to set the call to get ``Error in `across()`:``

Reply:
Definitely on my TODO list 😄